index.html

```diff
@@ -24,6 +24,214 @@ <h2 class="support">I’am a Software Developer</h2>
+                          <div class="tag">
+                            <div class="ruby">Ruby</div>
+                        <div class="bootstrap"> bootstrap</div>
+                        <div class="html">html</div>
+                          </div>
+                        </div>
+                              
+                            </div>
+                            <div class="cardwork6">
+                              <div class="container">
+                                <div class="action">
+                                <button class="buton1" type="button"> See Project</button>
+                              </div>
+                             <div class="supporttext">
+                              <p class="supporttext1">A daily selection of privately personalized reads; 
+                                no accounts or sign-ups required. has been the industry's standard</p>
+                                </div>
+                                <div class="titlpost">Profesional Art Printing Data
+                              </div>
+                              <div class="tag">
+                                <div class="ruby"> Ruby</div>
+                                <div class="bootstrap"> bootstrap</div>
+                                <div class="html">html</div>
+                              </div>
+                            </div>
+                                  
+                                </div>
+
+         </div>
+         <div class="featuredimg">
+          <div class="container1">
+            <div class="action1">
+            <button class="buton2" type="button"> See Project</button>
+          </div>
+          <div class="tag1">
+            <div class="ruby1"> Ruby</div>
+            <div class="bootstrap1"> bootstrap</div>
+            <div class="html1"> html</div>
+            <div class="css1"> css</div>
+          </div>
+         <div class="supporttext11">
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. has been the industry's standard</p>
+            </div>
+            <div class="titlpost11">Multi-Post Stories
+          </div>
+         </div>
+         <div class="imgplace">
+
+        </div>
+         </div>
+        </div>
+    <div class="title-work">
+      <div class="h2r"></div>
+      <div class="titl">My Recent Works </div>
+    </div>   
+  </div>
+  <div class="aboutdiv">
+    <div class="action2">
+      <button class="buton3" type="button"> Get My Resume</button>
+    </div>
+  <div class="descrpt">
+    Hello I’m a software developer! 
+    I can help you build a product , 
+    feature or website Look through some of my work
+     and experience! If you like what you see and 
+     have a project you need coded, don’t hestiate to contact me.
+
+  </div>
+  <div class="prmtxt">
+    About me
+  </div>
+  <div class="grpimg">
+    <img src="image/Frame.png" alt="vecto image">
+  </div>
```

Comment:
- [ ] Kindly don't use `img` tag here, decorations are always backgrounded images, not `img` tags. Kindly use `background-image: url("Image url");` in your CSS.